src/platforms/java/common/performance/instrumentation/opentelemetry.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: OpenTelemetry Support
+sidebar_order: 20
+description: "Using OpenTelemetry with Sentry Performance."
+---
+
+<Include name="alpha-note.mdx" />
+
+You can configure your [OpenTelemetry SDK](https://opentelemetry.io/) to send traces and spans to Sentry.
+
+There are multiple ways of combining OpenTelemetry and Sentry.
+
+## Using `sentry-opentelemetry-agent` with auto initialization
+
+By default, if you use `sentry-opentelemetry-agent`, it will look for `SENTRY_DSN` and `SENTRY_PROPERTIES_FILE` environment variables to be defined. If either of those is set, `sentry-opentelemetry-agent` will initialize Sentry automatically and you don't have to perform any setup work besides configuring your `DSN` and `tracesSampleRate`.
+
+### Install
+
+<PlatformContent includePath="performance/opentelemetry-install/with-java-agent" />
+
+### Usage
+
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/with-auto-init" />
+
+### Debugging
+
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/with-auto-init/debugging" />
+
+### Getting rid of exporter error messages
+
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/exporter-messages" />
+
+## Using `sentry-opentelemetry-agent` without auto initialization
+
+You may also disable automatic initialization of Sentry in `sentry-opentelemetry-agent` by setting `SENTRY_AUTO_INIT=false` as environment variable. This means you will either have to use another Sentry integration that performs initialization, for example Spring Boot, or initialize Sentry manually.
+
+### Install
+
+<PlatformContent includePath="performance/opentelemetry-install/with-java-agent" />
+
+### Usage
+
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/without-auto-init" />
+
+### Debugging
+
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/without-auto-init/debugging" />
+
+### Getting rid of exporter error messages
+
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/exporter-messages" />
+
+## Using OpenTelemetry without any Java Agent
+
+When manually initializing OpenTelemetry you have to register a few Sentry classes.
+
+### Install
+
+<PlatformContent includePath="performance/opentelemetry-install/without-java-agent" />
+
+### Usage
```

Comment:
Would be great to add an introductory sentence to introduce the user to the section and make the page easier to read. Please avoid stacking headings without text in-between.